### DISCUSSION TITLE
Why does [@types/jest Package] Version Not Match Jest Package Version

### DISCUSSION THREAD

**Discussion by estrudels** [DefinitelyTyped/DefinitelyTyped#67029] [Issues with a @types package] [answered]
[@types/jest]

I am curious why `@types/jest` package version is not "synced" with the  'jest" package version
* Currently `@types/jest` is at version `29.5.5` as of 10/11/2023
* Currently `jest` is at version `29.7.0` as of 10/11/2023

For reference
* NPM :: jest :: https://www.npmjs.com/package/jest
* NPM :: @types/jest :: https://www.npmjs.com/package/@types/jest
* GitHub :: ts-jest :: https://kulshekhar.github.io/ts-jest/

I would appreciate some feedback from the community, thank you 😃💛

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
It looks like Jest bundles its own types, so the `@types` package shouldn't be needed:

![Screenshot of the Jest package on npm](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/13910387/e8b36cbb-f568-4580-b837-81ae87838876)

**Reply by estrudels** [reply]
@MysteryBlokHed Thank you for your response and feedback it helped clarify the situation and understand we can just rely on the `jest` package for the types

**Reply by jakebailey** [reply]
IIRC `@types/jest` is still around to get the globals, though my personal opinion is that importing `@jest/globals` is superior.
